plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -576,6 +576,7 @@ func isKnownCounterDataError(err error) bool {
 		pdhErr.ErrorCode == PdhCalcNegativeDenominator ||
 		pdhErr.ErrorCode == PdhCalcNegativeValue ||
 		pdhErr.ErrorCode == PdhCstatusInvalidData ||
+		pdhErr.ErrorCode == PdhCstatusNoInstance ||
 		pdhErr.ErrorCode == PdhNoData) {
 		return true
 	}
```

Comment:
This really should be converted to a switch statement one day.... But maybe not today. ;-)